checks/evaluation/fuzzing_test.go

```diff
@@ -47,6 +47,26 @@ func TestFuzzing(t *testing.T) {
 						Probe:   "fuzzedWithGoNative",
 						Outcome: finding.OutcomeNegative,
 					},
+					{
+						Probe:   "fuzzedWithPythonAtheris",
+						Outcome: finding.OutcomeNegative,
+					},
+					{
+						Probe:   "fuzzedWithCLibFuzzer",
+						Outcome: finding.OutcomeNegative,
+					},
+					{
+						Probe:   "fuzzedWithCppLibFuzzer",
+						Outcome: finding.OutcomeNegative,
+					},
+					{
+						Probe:   "fuzzedWithRustCargofuzz",
+						Outcome: finding.OutcomeNegative,
+					},
+					{
+						Probe:   "fuzzedWithJavaJazzerFuzzer",
+						Outcome: finding.OutcomeNegative,
+					},
 					{
```

Comment:
How about swift and javascript too?

Reply:
javascript has some code already to look for `fast-check`, which was added as part of #3073

Reply:
I suggest doing Javascript in a follow-up PR because the logic of the fuzzing check isn't set up well for having multiple different fuzzers in a given language. I think it would be nice to avoid adding more complexity to this PR.

Reply:
Added Swift support. Added jazzer.js as an issue for now and will address in a follow-up PR https://github.com/ossf/scorecard/issues/3491